character-controller.js

```diff
@@ -434,6 +440,20 @@ class PlayerBase extends THREE.Object3D {
       _emitEvents();
     }
   }
+  setTarget(target) { // set both head and eyeball target;

```

Comment:
I think in the future we want to improve this to split the eyes and head into separate procedural animations, so the character:

1) moves their head a little around the target area
2) looks away sometimes